src/franka_plan_runner.cc

```diff
@@ -115,14 +126,13 @@ void FrankaPlanRunner::SetCollisionBehaviorSafetyOn(franka::Robot& robot) {
                              + utils::RobotModeToString(mode)
                              + " cannot change collision behavior!");
   }
-  robot.setCollisionBehavior({{40.0, 40.0, 36.0, 36.0, 32.0, 28.0, 24.0}},
-                             {{40.0, 40.0, 36.0, 36.0, 32.0, 28.0, 24.0}},
-                             {{40.0, 40.0, 36.0, 36.0, 32.0, 28.0, 24.0}},
-                             {{40.0, 40.0, 36.0, 36.0, 32.0, 28.0, 24.0}},
-                             {{40.0, 40.0, 40.0, 50.0, 50.0, 50.0}},
-                             {{40.0, 40.0, 40.0, 50.0, 50.0, 50.0}},
-                             {{40.0, 40.0, 40.0, 50.0, 50.0, 50.0}},
-                             {{40.0, 40.0, 40.0, 50.0, 50.0, 50.0}});
+
+  // Changes the collision behavior. Forces or torques above the upper
+  // threshold are registered as collision and cause the robot to stop moving.
+  robot.setCollisionBehavior(upper_torque_threshold_, upper_torque_threshold_,
```

Comment:
Could you add a comment about why we need to pass this four times?

Reply:
Yeah.